docs/Developers/Development_Setup.rst

```diff
@@ -11,8 +11,29 @@ Virtualenv
 
 We highly recommend installing coala in a virtualenv for development. This
 will allow you to have a contained environment in which to modify coala,
-separate from any other installation of coala that you may not want to
-break. For more information about virtualenvs, please refer to the
+separate from any other installation of coala that you may not want to break.
```

Comment:
it would be helpful if you could tell me what exactly to add as why we use it? also you mentioned this in https://github.com/coala/documentation/pull/381 so do I add them to both?